File: Java/프로그래머스/0/120891. 369게임/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;
        String s = String.valueOf(order);
        
        for(int i=0; i<s.length(); i++) {
            char a = s.charAt(i);
            if(a == '3' || a == '6' || a == '9') {
                answer ++;
            }
        }
        return answer;
    }
}

File: JavaScript/프로그래머스/0/120891. 369게임/369게임.js
function solution(order) {
    var answer = 0;
    order = order.toString();
    for(let i=0; i<order.length; i++) {
        if(order.charAt(i) == '3' || order.charAt(i) == '6'|| order.charAt(i) == '9')
            answer++;
    }
    return answer;
}